.travis.yml

```diff
@@ -11,13 +11,6 @@ dist: trusty
 osx_image: xcode8.3
 
 before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-      brew update;
-      brew install openssl;
-      mkdir -p /usr/local/lib;
-      ln -s /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib /usr/local/lib/;
-      ln -s /usr/local/opt/openssl/lib/libssl.1.0.0.dylib /usr/local/lib/;
-    fi
```

Comment:
this is no longer required as `download.sh` installs a beta version of PowerShell